### PR TITLE
Use default timezone as 'Asia/Tokyo'

### PR DESCRIPTION
**Pull request by kachick**

### CODE REVIEW

File: lib/sbpayment/time.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module Sbpayment
+  module Time
+    class << self
+      def format_current_time
+        in_jst{ ::Time.now }.strftime('%Y%m%d%H%M%S')
+      end
+
+      def in_jst
+        ENV['TZ'], old = 'JST', ENV['TZ']
```
Comment (banyan) [inline lib/sbpayment/time.rb]: I tested on production server.

```rb
> ENV['TZ'] = 'JST'
=> "JST"
> Time.now
=> 2016-01-12 07:19:08 +0000
```

It didn't change, so do we need to do this?

```rb
> ENV['TZ'] = 'Asia/Tokyo'
=> "Asia/Tokyo"
> Time.now
=> 2016-01-12 16:18:43 +0900
```

Comment (kachick) [inline lib/sbpayment/time.rb]: ref: https://ja.wikipedia.org/wiki/Tz_database
```
よくある間違いはTZにJSTと指定してしまうことである。これだとTZにUTCを指定した場合と同じになってしまう。POSIXのTZ環境変数の表記に合わせるならば、JST-9と指定しなければならない（符号の正負に注意せよ）。tz databaseの作者はtz databaseが利用できる環境ならば、このようなPOSIX表記を利用せずとも、最初に挙げたAsia/TokyoのようなArea/Location表記を利用した方が誤りを軽減できると述べている[3]。
```

:bow:

File: spec/sbpayment/time_util_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  around do |example|
+    ENV['TZ'], old = 'UTC', ENV['TZ']
+    example.run
+    ENV['TZ'] = old
+  end
+
+  describe '.format_current_time' do
+    subject { described_class.format_current_time }
+
+    it { is_expected.to eq((Time.now + 9 * 60 * 60).strftime Sbpayment::TimeUtil::STRFTIME_FORMAT) } # TODO Consider to use `timecop` for stable results
```
Comment (kachick) [inline spec/sbpayment/time_util_spec.rb]: @banyan What do you think to use timecop in test?

Comment (banyan) [inline spec/sbpayment/time_util_spec.rb]: yes, it's okay :+1:

Comment (kachick) [inline spec/sbpayment/time_util_spec.rb]: Updated in https://github.com/quipper/sbpayment.rb/commit/c4bcf02e491b68e8f59eac4371636b6b0886883e :watch: :cop:

**Comment by banyan** [comment]
released as https://github.com/quipper/sbpayment.rb/releases/tag/v0.5.0

**Pull request merged**